options/linux/include/scsi/sg.h

```diff
@@ -42,6 +47,29 @@ struct sg_io_hdr {
     unsigned int info;
 };
 
+struct sg_scsi_id {
```

Comment:
is this sort of stuff not provided by the linux-headers option?

Reply:
While we don't have a `linux-headers` option any more, these header files are explicitly not provided by the kernel.

Reply:
in that case we would indeed have to provide these ourselves. I think that addresses all my concerns with this PR